articles/time-management.md

```diff
@@ -0,0 +1,40 @@
+---
+authorName: Christian Abegg
+authorGithubUsername: abeggchr
+---
+# Start using a time management technique today
```

Comment:
I propose that we use capitalized captions, as usual in English
so e.g. "Use a Time Management Technique - Now!" 
(und vielleicht gefällt Dir ja der leicht angepasste Vorschlag?)
see e.g. http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html
zumindest jene Regeln, wo alle Styles sich einig sind, sollten wir einhalten. Der Rest ist mir egal, da ich keine dogmatische Diskussion anreissen möchte oder mit unnötigem Zeit verlieren will.